Automattic/vip-go-mu-plugins
# Update WordPress Importer to 0.7.0

## Description

This PR Updates the [WordPress Importer Plugin](https://wordpress.org/plugins/wordpress-importer/) from 0.6.4 to the latest version which supports PHP 7.4 and has other improvements. 

Changelog: 

* Update minimum WordPress requirement to 3.7 and ensure compatibility with PHP 7.4.
* Fix bug that caused not importing term meta.
* Fix bug that caused slashes to be stripped from imported meta data.
* Fix bug that prevented import of serialized meta data.
* Fix file size check after download of remote files with HTTP compression enabled.
* Improve accessibility of form fields by adding missing labels.
* Improve imports for remote file URLs without name and/or extension.
* Add support for `wp:base_blog_url` field to allow importing multiple files with WP-CLI.
* Add support for term meta parsing when using the regular expressions or XML parser.
* Developers: All PHP classes have been moved into their own files.
* Developers: Allow to change `IMPORT_DEBUG` via `wp-config.php` and change default value to the value of `WP_DEBUG`.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Go to `wp-admin` > `Tools` > `Import` > `WordPress`
1. Import a WordPress XML file like the [Theme Unit Test](https://raw.githubusercontent.com/WPTT/theme-unit-test/master/themeunittestdata.wordpress.xml)
1. Verify the import is complete and there are no errors.
